src/pages/Environments.js

```diff
@@ -877,11 +882,19 @@ const Environments = () => {
   ])
 }
 
+// Temporary export here for ease of access to it when using the above component.
+export { ajaxContext }
```

Comment:
Nit: As an alternative to importing `ajaxContext` and exporting it, this could use `export from`, which does not make `ajaxContext` available within this module and thus encourages any future code to use `useAjax` instead.

```js
export { ajaxContext } from 'src/libs/ajax'
```

https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#re-exporting_aggregating

Reply:
Yes! This is better!